openmmml/models/anipotential.py

```diff
@@ -57,22 +57,32 @@ class ANIPotentialImpl(MLPotentialImpl):
     def __init__(self, name):
         self.name = name
 
+
     def addForces(self,
                   topology: openmm.app.Topology,
                   system: openmm.System,
                   atoms: Optional[Iterable[int]],
                   forceGroup: int,
                   filename: str = 'animodel.pt',
+                  implementation : str = 'nnpops',
+                  overwrite_periodicity : Optional[bool]=False,
                   **args):
+        """
+        overwrite_periodicity : whether to overwrite the periodicity given by `topology` or `system` to its opposite; this is primarily for 
+            testing purposes
```

Comment:
If this was only added for testing, should we remove it?  Or is it something users would actually want to do?

Reply:
will remove it. we don't need anymore